.rubocop_todo.yml

```diff
@@ -0,0 +1,326 @@
+# This configuration was generated by `rubocop --auto-gen-config`
+# on 2015-04-27 23:58:52 +0200 using RuboCop version 0.30.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
```

Comment:
The hard truth is that people still run Ruby 1.8 in production. So it has to be supported in this project.